app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
+  end
+
+  def update 
+    @game = Game.find(params[:id])
+    @game.update_attributes(black_player_id: current_user.id)
+    redirect_to game_path(@game.id)
+  end
+
+  private
+
+  def game_params
+    params.require(:game).permit(:name, :black_player_id)
```

Comment:
Well to some extent it depends on the use case that `update` is meant to handle. My understanding from reading through the code is that you intended to have it support the scenario where a second player is joining a game with an open seat. If that's the case, and we go with the simple approach of always assigning the first user to be the white player, then `black_player_id` still needs to be updated as part of the action.

However, it doesn't need to be part of `game_params`. The point of `params.require` and `params.permit` is to whitelist certain values that come through the params hash for use in record creation/modification. For security reasons, they can't be used this way by default: http://guides.rubyonrails.org/action_controller_overview.html#strong-parameters. Since the player ID is coming from Devise here rather than from the params hash, there's no need for it to be whitelisted.